topics/user_interface_components/status_bar_widgets.md

```diff
@@ -0,0 +1,124 @@
+[//]: # (title: Status Bar Widgets)
+
+<!-- Copyright 2000-2022 JetBrains s.r.o. and other contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file. -->
+
+The status bar widget implementation is used to add new widgets to the
+[status bar](https://www.jetbrains.com/help/idea/guided-tour-around-the-user-interface.html#status-bar).
+
+Widgets on the status bar are small interface elements that allow you to provide the user important information about the current file or the project.
+For example, the widget showing the encoding of the current file or the current VCS branch of the project.
+
+The starting point for extending the status bar with new widgets is the
+[`StatusBarWidgetFactory`](upsource:///platform/platform-api/src/com/intellij/openapi/wm/StatusBarWidgetFactory.java)
+interface, which is registered in the `com.intellij.statusBarWidgetFactory` extension point.
+
+To reuse the IntelliJ Platform implementation, you can extend the
+[`StatusBarEditorBasedWidgetFactory`](upsource:///platform/platform-api/src/com/intellij/openapi/wm/impl/status/widget/StatusBarEditorBasedWidgetFactory.java)
+class.
+
+> Note that widgets inherited from this class will only work when at least one file is open.
```

Comment:
If the previous comment is accepted, I suggest removing this callout completely as it seems redundant.